201-vmss-ubuntu-web-ssl/azuredeploy.json

```diff
@@ -386,7 +383,7 @@
                   "autoUpgradeMinorVersion": false,
                   "settings": {
                     "fileUris": [
-                      "[concat(parameters('_artifactsLocation'), '/', parameters('scriptFileName'), parameters('_artifactsLocationSasToken'))]"
+                      "[uri(parameters('_artifactsLocation'), concat('/', parameters('scriptFileName'), parameters('_artifactsLocationSasToken')))]"
```

Comment:
remove the extra slash, e.g.

"[uri(parameters('_artifactsLocation'), concat(parameters('scriptFileName'), parameters('_artifactsLocationSasToken')))]"